docs/extend/modular-resources/_index.md

```diff
@@ -279,6 +282,55 @@ func init() {
 }
 ```
 
+<file>mymodule.go</file>
+
+``` go {class="line-numbers linkable-line-numbers"}
+// Package main is a module which serves the mybase custom model type.
+package main
+
+import (
+    "context"
+
+    "github.com/edaniels/golog"
+
+    "go.viam.com/rdk/components/base"
+    "go.viam.com/rdk/module"
+    "go.viam.com/utils"
+
+    // NOTE: You must update the following line to import your local package "mybase"
+    "go.viam.com/rdk/examples/customresources/models/mybase"
+)
+
+func main() {
+    // NewLoggerFromArgs will create a golog.Logger at "DebugLevel" if
+    // "--log-level=debug" is the third argument in os.Args and at "InfoLevel"
```

Comment:
Not sure about this "third argument" part. Shouldn't the argument flags work in any order? Hopefully it'll create the logger at the right debug level if that arg is present anywhere in `os.Args`.

Reply:
@penguinland  I adapted this sample from the RDK demos and this note came w/ --> I agree it seems misleading if the flag is what's triggering 
Making this change here